include/z5/file.hxx

```diff
@@ -9,7 +9,7 @@ namespace z5 {
     inline void createFile(const handle::File & file, const bool isZarr=true) {
         file.createDir();
         if(isZarr) {
-            Metadata fmeta;
+            Metadata fmeta(isZarr);
```

Comment:
In the last PR, we have updated the metadata so that it contains the n5 version string as well, see [here](https://github.com/constantinpape/z5/blob/33f506f0b410157bfad64e95e6be88bb156e9e1f/include/z5/metadata.hxx#L24). Thus we don't need to check here any more if the file is zarr and n5, because this will be done in `writeMetadata` in any case. So instead of
```
if(isZarr) {
...
} else {
...
}
``` 
we can just do
```
Metadata fmeta(isZarr);
writeMetadata(file, fmeta);
```
Could you change this please?